src/m3/test/constraints.test.ts

```diff
@@ -0,0 +1,94 @@
+import {
+    assertEquals
+} from "https://deno.land/std@0.160.0/testing/asserts.ts"
+
+import {
+    Annotation,
+    Concept,
+    ConceptInterface,
+    Containment,
+    Metamodel,
+    Multiplicity,
+    Property,
+    Reference
+} from "../types.ts"
+import {issuesMetamodel} from "../constraints.ts"
+
+
+Deno.test("constraints (lioncore)", async (tctx) => {
+
+    await tctx.step("check that a ConceptInterface only has derived features", async () => {
```

Comment:
It would be nice to find a way to share the tests between the different implementations

Reply:
Yes! For that we'd probably need a language-**a**specific description of the tests...maybe in the form of a model? ;)